deploy/charts/cert-manager/values.docs.yaml

```diff
@@ -0,0 +1,713 @@
+- default: false
```

Comment:
It'd be nice to add a comment at the top of this file explaining that it's auto-generated by `github.com/amurant/helm-jsonschema-gen/cmd/helm-jsonschema-gen` and explaining that it is consumed by `tem`